roles/infrastructure/tasks/teardown_aws_network.yml

```diff
@@ -16,9 +16,7 @@
 
 - name: Tear down AWS Security Groups associated with enviroment and datalake
   when: 
-    - infra__teardown_deletes_network 
```

Comment:
If the flag is set NOT touch any network, we still go ahead and delete the SGs? Why must we delete always?